crates/ruff_workspace/src/options.rs

```diff
@@ -1088,6 +1092,35 @@ impl Flake8BuiltinsOptions {
         }
     }
 }
+
+#[derive(
+    Debug, PartialEq, Eq, Default, Serialize, Deserialize, OptionsMetadata, CombineOptions,
+)]
+#[serde(deny_unknown_fields, rename_all = "kebab-case")]
+#[cfg_attr(feature = "schemars", derive(schemars::JsonSchema))]
+pub struct Flake8CognitiveComplexityOptions {
+    /// The maximum cognitive complexity to allow before triggering `CCR001` errors.
+    #[option(
+        default = "7",
```

Comment:
How did you decide on the default? Is it the same as the upstream rule uses?

Reply:
Correct! I copied the default max [ref](https://github.com/Melevir/flake8-cognitive-complexity/blob/d88502e1abdcc4540c27dac430c616ae6ba1eb9b/flake8_cognitive_complexity/checker.py#L10).

Though the author thinks it should be 15 but also states that there perhaps shouldn't be a default... [ref](https://stackoverflow.com/a/45084107)